docs/advanced.rst

```diff
@@ -136,7 +136,7 @@ Create your own persistence class, see the example below:
 
 Your custom persister must implement both ``load_cassette`` and ``save_cassette``
 methods.  The ``load_cassette`` method must return a deserialized cassette or raise
-``ValueError`` if no cassette is found.
+``CassetteNotFoundError`` if no cassette is found.
```

Comment:
This should mention `CassetteDecodeError` also somehow or no one will know about that exception with reading the internal code.

Reply:
Done! Feel free to change the wording further.